reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Draft Orders API
+  x-logo:
+    url: "https://andrewjtech123.github.io/imagehost/shopify_logo_white.png"
+    backgroundColor: "#7ab55c"
+  description: >
+    # Draft Orders API 
+      
+      A [Draft Order](https://help.shopify.com/manual/orders/create-orders) represents the status of an order before the customer has completed the checkout process. Using the Draft Orders API you can create a draft order and transition it to a completed order with great flexibility.  The following flows are supported:
```

Comment:
This top paragraph is great!

"before the customer has completed the checkout process" implies there is always a checkout, I'd be a bit more generic and say something that means "before the order has been completed" perhaps? Before the price, and line items, and exact contents of the order are finalized? the point is, its a draft if you can still change what the contents of the order are and what the exact pricing and payment method is.